CONTRIBUTING.md

```diff
@@ -0,0 +1,135 @@
+## Contributing In General
+Our project welcomes external contributions. If you have an itch, please feel
+free to scratch it.
+
+To contribute code or documentation, please submit a [pull request](https://github.com/IBM/compliance-trestle/pulls).
+
+A good way to familiarize yourself with the codebase and contribution process is
+to look for and tackle low-hanging fruit in the [issue tracker](https://github.com/IBM/compliance-trestle/issues).
+Before embarking on a more ambitious contribution, please quickly [get in touch](MAINTAINERS.md) with us.
+
+**Note: We appreciate your effort, and want to avoid a situation where a contribution
+requires extensive rework (by you or by us), sits in backlog for a long time, or
+cannot be accepted at all!**
+
```

Comment:
Reference the code of conduct. See the auditree contributing file(complianceascode/auditree-framework).